src/auth/jwt-auth.guard.ts

```
@@ -0,0 +1,30 @@
+
+import {
```

ESLint

[error] Delete `⏎`

(prettier/prettier)